.github/workflows/avm.template.module.yml

```
@@ -81,6 +104,7 @@ jobs:
       (fromJson(inputs.workflowInput)).deploymentValidation == 'true' && needs.job_module_static_validation.result != 'failure'
```

We should block next jobs on psrule reliability failures
```suggestion
      (fromJson(inputs.workflowInput)).deploymentValidation == 'true' && needs.job_module_static_validation.result != 'failure' && needs.job_psrule_test_waf_reliability != 'failure'
```